docroot/modules/custom/layout_builder_custom/js/layout_builder_custom.overrides.js

```diff
@@ -26,11 +26,12 @@
 
           let justCreated = true;
 
-          if (Drupal.offCanvas.isOffCanvas($element) && $element.find('.layout-selection').length === 1) {
+          // Removed $element.find('.layout-selection').length check.
```

Comment:
I think a comment would be more useful here if it explained what is happening (if its not obvious). The relevance of this particular comment will be gone pretty much as soon as this gets merged.